src/RwaConduit.sol

```diff
@@ -22,3 +21,53 @@ contract RwaConduit {
         dai.transfer(to, dai.balanceOf(address(this)));
     }
 }
+
+contract RwaRoutingConduit {
+    // --- auth ---
+    mapping (address => uint) public wards;
+    mapping (address => uint) public can;
+    function rely(address usr) external auth { wards[usr] = 1; }
+    function deny(address usr) external auth { wards[usr] = 0; }
+    modifier auth {
+        require(wards[msg.sender] == 1, "RwaConduit/not-authorized");
+        _;
+    }
+    function hope(address usr) external auth { can[usr] = 1; }
+    function nope(address usr) external auth { can[usr] = 0; }
+    modifier operator {
+        require(can[msg.sender] == 1, "RwaConduit/not-operator");
+        _;
+    }
+
+    GemLike public gov;
+    GemLike public dai;
+
+    address public to;
+    mapping (address => uint) public bud;
+
+    constructor(address _gov, address _dai) public {
+        wards[msg.sender] = 1;
+        gov = GemLike(_gov);
+        dai = GemLike(_dai);
+    }
+
+    // --- administration ---
+    function kiss(address who) public auth {
```

Comment:
This can be changed to `external` to save memory copy gas. Same with the other methods.